Source/Scene/Model.js

```diff
@@ -4799,13 +4818,15 @@ function modifyShaderForClippingPlanes(
   shader +=
     "uniform highp sampler2D gltf_clippingPlanes; \n" +
     "uniform mat4 gltf_clippingPlanesMatrix; \n" +
+    "uniform mat4 gltf_normalClippingPlanesMatrix; \n" +
```

Comment:
The normal matrix should probably be a `Matrix3` everywhere. This keeps it consistent with `czm_normal` which is also a `Matrix3`.